e2e-tests/fixtures.ts

```diff
@@ -55,7 +64,8 @@ base.beforeAll(async () => {
 
   await page.waitForLoadState('domcontentloaded')
   await page.waitForFunction('window.document.title != "Loading"')
-  await page.waitForSelector('text=This is a demo graph, changes will not be saved until you open a local folder')
+  // NOTE: The following ensures first start.
+  // await page.waitForSelector('text=This is a demo graph, changes will not be saved until you open a local folder')
```

Comment:
I think we can wait for the loading spinner here. In https://github.com/pengx17/logseq-publish/commit/cdce58f61ec8067f2b7c4041d5040930254ee23e, I found that load a local graph could fail if the app is still initializing the default demo graph.

Reply:
good point